dev-util/gnome-builder/gnome-builder-45_rc.ebuild

```diff
@@ -0,0 +1,272 @@
+# jhbuild support
+# rust support via rust-analyzer; Go via go-langserver
+# autotools stuff for autotools plugin; gtkmm/autoconf-archive for C++ template
+# gjs/gettext/mono/PHPize stuff, but most of these are probably installed for other reasons anyways, when needed inside IDE
+# stylelint for stylesheet (CSS and co) linting
+# gvls for vala language-server integration
+
+llvm_check_deps() {
+	has_version "sys-devel/clang:${LLVM_SLOT}"
+}
+
+pkg_setup() {
+	python-single-r1_pkg_setup
+	use clang && llvm_pkg_setup
+}
+
+src_configure() {
+	local emesonargs=(
+		-Dchannel=other
+
+		-Ddevelopment=false
+		-Dtracing=false
+		-Dprofiling=false # not passing -pg to CFLAGS
+		-Dtcmalloc=false
+		-Dwith_safe_path=''
+		-Dgnome_sdk_version=master
+		$(meson_use doc help)
+		$(meson_use gtk-doc docs)
+		-Dnetwork_tests=false
+		-Dctags_path=''
+		$(meson_feature webkit)
+
+		-Dplugin_autotools=true
+		-Dplugin_bash_language_server=true
+		-Dplugin_blueprint=true
+		-Dplugin_buildstream=true
+		-Dplugin_c_pack=true
+		-Dplugin_cargo=true
+		$(meson_use clang plugin_clang)
+		$(meson_use clang plugin_clangd)
+		$(meson_use clang plugin_clang_format)
+		-Dplugin_cmake=true
+		-Dplugin_codesearch=true
+		-Dplugin_codeshot=true
+		-Dplugin_codespell=true
+		-Dplugin_code_index=true
+		-Dplugin_copyright=true
+		-Dplugin_ctags=true
+		-Dplugin_deviced=false # libdeviced not packaged?
+		$(meson_use d-spy plugin_dspy)
+		-Dplugin_dub=true
+		-Dplugin_editorconfig=true
+		-Dplugin_eslint=true
+		-Dplugin_file_search=true
+		$(meson_use flatpak plugin_flatpak)
+		-Dplugin_gdb=true
+		-Dplugin_gdiagnose=true
+		-Dplugin_gettext=true
+		$(meson_use git plugin_git)
+		-Dplugin_glsl_language_server=true
+		-Dplugin_golang=true
+		-Dplugin_gopls=true
+		-Dplugin_gradle=true
+		-Dplugin_grep=true
+		-Dplugin_hadolint=true
+		-Dplugin_html_completion=true
+		$(meson_use webkit plugin_html_preview)
+		-Dplugin_intelephense=true
+		-Dplugin_jdtls=true
+		-Dplugin_jedi_language_server=false
+		-Dplugin_jhbuild=true
+		-Dplugin_lua_language_server=true
+		-Dplugin_make=true
+		-Dplugin_make_templates=true
+		$(meson_use webkit plugin_markdown_preview)
+		$(meson_use webkit plugin_markdown_indenter)
+		-Dplugin_maven=true
+		-Dplugin_meson=true
+		-Dplugin_meson_templates=true
+		-Dplugin_modelines=true
+		-Dplugin_mono=true
+		-Dplugin_newcomers=true
+		-Dplugin_notification=true
+		-Dplugin_npm=true
+		-Dplugin_phpize=true
+		-Dplugin_podman=true
+		-Dplugin_pygi=true
+		-Dplugin_python_lsp_server=true
+		-Dplugin_qemu=true
+		-Dplugin_quick_highlight=true
+		-Dplugin_retab=true
+		-Dplugin_rstcheck=true
+		-Dplugin_rubocop=true
+		-Dplugin_rust_analyzer=false # rust-analyzer not packaged
+		-Dplugin_serve_d=true
+		-Dplugin_shellcheck=true
+		-Dplugin_shellcmd=true
+		-Dplugin_sourcekit_lsp=true
+		$(meson_use spell plugin_spellcheck)
+		$(meson_use webkit plugin_sphinx_preview)
+		-Dplugin_stylelint=true
+		-Dplugin_swift=true
+		-Dplugin_swift_mesonlsp=true
```

Comment:
The diff to `meson_options.txt` was

```diff
@@ -88,6 +89,7 @@ option('plugin_stylelint', type: 'boolea
 option('plugin_swift', type: 'boolean')
 option('plugin_swiftformat', type: 'boolean')
 option('plugin_swiftlint', type: 'boolean')
+option('plugin_swift_mesonlsp', type: 'boolean')
 option('plugin_sysprof', type: 'boolean')
 option('plugin_todo', type: 'boolean')
 option('plugin_ts_language_server', type: 'boolean')
```

The diff you made to the ebuild was
```diff
@@ -221,6 +217,7 @@
 		$(meson_use webkit plugin_sphinx_preview)
 		-Dplugin_stylelint=true
 		-Dplugin_swift=true
+		-Dplugin_swift_mesonlsp=true
 		-Dplugin_swiftformat=true
 		-Dplugin_swiftlint=true
 		$(meson_use sysprof plugin_sysprof)
```

Please put the new option in the same order as in `meson_options.txt`.